### PR TITLE
fix in diagnose cluster permissions: add getting deployments and namespaces

### PR DESCRIPTION
**Pull request by tomer-shefler**
fixing: https://github.com/submariner-io/subctl/issues/129

### CODE REVIEW

**Review by mkolesnik — CHANGES_REQUESTED** [review]
The change is good but please update the commit message with **why** it's needed, e.g. "The diagnose needs get/list capability for namespaces to ..."

**Comment by dfarrell07** [comment]
> The change is good but please update the commit message with **why** it's needed, e.g. "The diagnose needs get/list capability for namespaces to ..."

https://submariner.io/development/code-review/#document-why-in-commit-messages

**Comment by mkolesnik** [comment]
Please make sure to follow the commit message formatting  - https://submariner.io/development/code-review/#commit-message-formatting

File: pkg/embeddedyamls/yamls.go [hunk 1]
```diff
@@ -2790,13 +2790,15 @@ rules:
       - ""
     resources:
       - configmaps
+      - namespaces
```
Comment (mkolesnik) [inline pkg/embeddedyamls/yamls.go:2732]: This file is auto generated, you should be editing the source file `config/rbac/submariner-diagnose/role.yaml`

**Review by mkolesnik — CHANGES_REQUESTED** [review]

**Review by mkolesnik — APPROVED** [review]
Thanks

You need to shorten the commit title as it's too long, otherwise it's LGTM :)
https://github.com/submariner-io/submariner-operator/runs/7973475250?check_suite_focus=true#step:3:73

Apparently the lines are too long also.

You can run `make gitlint` locally before pushing the updated commit to make sure it's passing

**Review by tpantelis — APPROVED** [review]

**Pull request merged**